2017-07-24-tidbbestpractice.md

```diff
@@ -62,7 +62,7 @@ TiDB provides complete distributed transactions and the model has some optimizat
 	As distributed transactions need to conduct two-phase commit and the bottom layer performs Raft replication, if a transaction is very large, the commit process would be quite slow and the following Raft replication flow is thus struck. To avoid this problem, we limit the transaction size:
 
 	- Each Key-Value entry is no more than 6MB
-	- The total number of Key-Value entry is no more than 300,000 rows
+	- The total number of Key-Value entry is no more than 200 rows
```

Comment:
The limitation of the key-value entries count is still 300,000
Just update the best practice that it's better to keep each modification in a single transaction less than 200 rows